practice-one/src/js/services/apis.js

```diff
@@ -0,0 +1,79 @@
+import { ERROR_MESSAGES } from "../constants/errorMessages";
+
+const { API_BASE_URL } = process.env;
+
+async function get() {
```

Comment:
Should name it as getTodos or getTasks because this function is for fetching tasks only, it can not be reused for the other data